xla/service/gpu/dot_algorithm_support_test.cc

```diff
@@ -143,8 +153,25 @@ TEST_P(DotAlgorithmSupportTest, AlgorithmIsSupportedFromCudaCapability) {
       primitive_util::LowercasePrimitiveTypeName(params.output_storage_type),
       params.sizes.contracting_size, params.sizes.non_contracting_size);
 
-  if (GetCudaComputeCapability().IsAtLeast(params.min_cuda_capability.major,
-                                           params.min_cuda_capability.minor)) {
+  bool is_algorithm_supported = false;
+  if (IsCUDA()) {
```

Comment:
Please use this form:
```
if ( const auto* ccc = std::get_if<se::CudaComputeCapability>(GetGpuComputeCapability())) {
...
} else if ( const auto* rcc = std::get_if<se::CudaComputeCapability>(GetGpuComputeCapability())) {
...
}
```

This is both reasonably compact and doesn't assume that there is only Cuda and RoCM.

Reply:
Submitted a new commit